lib/iex/lib/iex/helpers.ex

```diff
@@ -393,7 +393,14 @@ defmodule IEx.Helpers do
   end
 
   @doc """
-  Prints information about the given data type.
+  Prints information about the data type of a given term.
+  
+  ## Examples
+
+      i([1,2,3])
+      i("Elixir rocks!")
+      i(:some_atom)
+      i(fn x -> x * 2 end)
```

Comment:
I would just pick one case that illustrates what the function does, and is not quiet long.
I think range is more complete than list, as it list more than one reference module